resources/lang/hu/validation.php

```diff
@@ -64,6 +66,10 @@
         'option' => 'szavazat',
         'alfonso_language' => 'Alfonsó nyelv',
         'alfonso_desired_level' => 'Alfonsó szint',
+        'semester_average' => array_fill(0, 10, 'korábbi szemeszter átlaga'), // from semester_average.0 to semester_average.9
+        'question_1' => "\"Honnan hallott a Collegiumról?\" kérdés",
```

Comment:
this is also implemented in one of my other PRs 🙃 . But maybe not all. Compare when merging.